src/test/java/picard/sam/FilterSamReadsTest.java

```diff
@@ -95,7 +95,7 @@ public void testPairedIntervalFilter(final String intervalFilename, final int ex
         // Build a sam file for testing
         final File inputSam = File.createTempFile("testSam", ".sam", TEST_DIR);
         inputSam.deleteOnExit();
-        final File sortedSamIdx = new File(TEST_DIR, inputSam.getName() + ".idx");
+        final File sortedSamIdx = new File(TEST_DIR, inputSam.getName() + ".bai");
```

Comment:
This test reliably failed to clean up after itself because of the wrong extension here.